gulp.d/tasks/build-preview-pages.js

```diff
@@ -116,6 +121,9 @@ module.exports =
                     pageModel.attributes = Object.entries({ ...attributes, ...componentVersion.asciidoc.attributes })
                       .filter(([name, val]) => name.startsWith('page-'))
                       .reduce((accum, [name, val]) => ({ ...accum, [name.substr(5)]: val }), {})
+
+                    // console.log(pageModel.attributes)
```

Comment:
🚗 Drive-by comment...
Guessing this is for testing things out, just reminder to remove once you're happy with it. 📆